int32 obj_type 
---
float64[] x
float64[] y